glotaran/io/preprocessor/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""Tools for data pre-processing."""
+from glotaran.io.preprocessor.pipeline import PreProcessingPipeline
```

Comment:
`CorrectBaselineValue` and `CorrectBaselineAverage` also be reexported from here for ease of use.

Reply:
Nobody should use them directly, it makes no sense. They are considered internal.

Reply:
Yeah nvm I'm just tired